aerospike-mixin/dashboards/aerospike-logs-overview.libsonnet

```
@@ -0,0 +1,32 @@
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+local logsDashboard = import 'github.com/grafana/jsonnet-libs/logs-lib/logs/main.libsonnet';
+{
+  grafanaDashboards+::
+    if $._config.enableLokiLogs then {
+      local aerospikeLogs =
+        logsDashboard.new(
+          'Aerospike logs',
+          datasourceName='loki_datasource',
+          datasourceRegex='',
+          filterSelector=$._config.filterSelector,
+          labels=['job', 'aerospike_cluster', 'instance', 'level'],
```

```suggestion
          labels=['job', 'aerospike_cluster', 'instance', 'context', 'level'],
```
Looks like context is another one useful label we can extract.